Rules/UseCompatibleCmdlets.cs

```diff
@@ -395,7 +479,7 @@ private bool IsValidPlatformString(string fileNameWithoutExt)
             psedition = null;
             psversion = null;
             os = null;
-            const string pattern = @"^(?<psedition>core|desktop)-(?<psversion>[\S]+)-(?<os>windows|linux|macos)$";
+            const string pattern = @"^(?<psedition>core.*|desktop)-(?<psversion>[\S]+)-(?<os>windows|linux|macos|nano|iot)$";
```

Comment:
`pattern` could be more descriptively named here. Also, if the value is constant it should ideally be at the top of a class